<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="saveModel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelModel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>461</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="metrics">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>561</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Metrics:
MAE: 
MSE:
RMSE: 
R-Squared: </string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите данные для обучения:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="teachModel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Обучить модель</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectModel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>441</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>12</number>
   </property>
   <item>
    <property name="text">
     <string>Модель 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Модель 2</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
